sdk/storage/inc/datalake/datalake_options.hpp

```diff
@@ -18,23 +20,42 @@ namespace Azure { namespace Storage { namespace DataLake {
    */
   struct ServiceClientOptions
   {
-    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> policies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerOperationPolicies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerRetryPolicies;
   };
 
   /**
    * @brief File system client options used to initalize DataLakeFileSystemClient.
    */
   struct FileSystemClientOptions
   {
-    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> policies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerOperationPolicies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerRetryPolicies;
   };
 
   /**
    * @brief Path client options used to initalize DataLakePathClient.
    */
   struct PathClientOptions
   {
-    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> policies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerOperationPolicies;
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> PerRetryPolicies;
+  };
+
+  /**
+   * @brief Specifies access conditions for a file system.
+   */
+  struct FileSystemAccessConditions : public LastModifiedTimeAccessConditions
+  {
+  };
+
+  /**
+   * @brief Specifies access conditions for a path.
+   */
+  struct PathAccessConditions : public LastModifiedTimeAccessConditions,
```

Comment:
Are you sure all path operations support these 5 access conditions?

Reply:
Yes.

Reply:
Let me clarify this. All path operation that supports `PathAccessConditions` supports these 5 access conditions.

Reply:
okay then